rpc/handle_get_peer_addresses.go

```diff
@@ -1,26 +1,26 @@
 package rpc
 
-import "github.com/kaspanet/kaspad/rpcmodel"
+import "github.com/kaspanet/kaspad/rpc/model"
 
 // handleGetPeerAddresses handles getPeerAddresses commands.
 func handleGetPeerAddresses(s *Server, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
-	peersState, err := s.cfg.addressManager.PeersStateForSerialization()
+	peersState, err := s.addressManager.PeersStateForSerialization()
 	if err != nil {
 		return nil, err
 	}
 
-	rpcPeersState := rpcmodel.GetPeerAddressesResult{
+	rpcPeersState := model.GetPeerAddressesResult{
```

Comment:
Please create ticket if there are any fields that are left TODO.

Reply:
There aren't.